### PR TITLE
[guitartricks] Add new extractor

### PR DESCRIPTION
**Pull request by thiemo**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)

### What is the purpose of your *pull request*?
- [x] New extractor

---

### Description of your *pull request* and other information

This PR adds support for downloading videos from [GuitarTricks](http://www.guitartricks.com).

### CODE REVIEW

File: youtube_dl/extractor/guitartricks.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        kwargs.setdefault('headers', {})['User-Agent'] = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/603.2.4 (KHTML, like Gecko) Version/10.1.1 Safari/603.2.4'
+        return super(GuitarTricksIE, self)._download_webpage(
+            *args, **compat_kwargs(kwargs))
+
+    def _real_initialize(self):
+        self._login()
+
+    def _login(self):
+        (username, password) = self._get_login_info()
+        return
```
Comment (dstftw) [inline youtube_dl/extractor/guitartricks.py]: ?!

File: youtube_dl/extractor/guitartricks.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        'md5': '09e89ad6c6a9b85b0471b80230995fcc',
+        'info_dict': {
+            'id': '5706o2esuo',
+            'ext': 'mp4',
+        },
+    }]
+
+
+    def _download_webpage(self, *args, **kwargs):
+        kwargs.setdefault('headers', {})['User-Agent'] = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/603.2.4 (KHTML, like Gecko) Version/10.1.1 Safari/603.2.4'
```
Comment (dstftw) [inline youtube_dl/extractor/guitartricks.py]: Remove.

**Comment by dstftw** [comment]
>- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

**Comment by thiemo** [comment]
revised entire extractor

**Comment by thiemo** [comment]
As I am new to PRs I am not quite sure what to do in order to re-start the review process. Do I have to somehow remove the "changes requested" label in order for a reviewer to continue the review process?

**Pull request closed without merging**